Project.toml

```diff
@@ -5,13 +5,14 @@ version = "0.3.1-DEV"
 
 [deps]
 FITSIO = "525bcba6-941b-5504-bd06-fd0dc1a4d2eb"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
The package itself must not depend on `Documenter`.  See https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#.travis.yml-Configuration-1 about how to configure Travis to produce documentation